todo_app/data/session_items.py

```diff
@@ -2,7 +2,8 @@
 
 _DEFAULT_ITEMS = [
     { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
+    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' },
+    { 'id': 3, 'main': 'Not Started', 'title': 'Added to test' }
```

Comment:
Why have you added an item to the list of default items? We want to allow the user to complete a form to add items dynamically. I think you should revert this change.